commons-profiler-api/src/main/java/ru/fix/commons/profiler/impl/ProfilerReporterImpl.java

```diff
@@ -121,7 +156,50 @@ private ProfilerCallReport buildReportAndReset(String name, SharedCounters count
                 .setReportingTime(elapsed)
 
                 .setMaxThroughputPerSecond(counters.getMaxThroughput().getMaxAndReset())
-                .setMaxPayloadThroughputPerSecond(counters.getMaxPayloadThroughput().getMaxAndReset());
+                .setMaxPayloadThroughputPerSecond(counters.getMaxPayloadThroughput().getMaxAndReset())
+
+                .setActiveCallsCount(counters.getActiveCallsCounter().sum())
+                .setActiveCallsMaxLatency(activeCallsMaxLatencyAndResetActiveCalls(counters));
+    }
+
+    private long activeCallsMaxLatencyAndResetActiveCalls(SharedCounters counters) {
+        ProfiledCallImpl longestCall = resetActiveCallsAndGetLongest(counters);
+        if (longestCall == null) {
+            return 0L;
+        }
+
+        return longestCall.timeFromCallStartInMs();
+    }
+
+    private ProfiledCallImpl resetActiveCallsAndGetLongest(SharedCounters counters) {
+        if (!enableActiveCallsMaxLatency.get() && !counters.getActiveCalls().isEmpty()) {
+            counters.getActiveCalls().reset();
+            return null;
```

Comment:
we have convention for returning NULLs from method:  you should at least declare in javadoc that method is Nullable or change signature to implicit Optional. Or use Kotlin =)